packages/palette/src/elements/Link/Link.tsx

```diff
@@ -6,6 +7,7 @@ import { boxMixin, BoxProps } from "../Box"
 type UnderlineBehaviors = "default" | "hover" | "none"
 
 export interface LinkProps extends BoxProps {
+  visitedColor?: Color
   hoverColor?: Color
   noUnderline?: boolean
```

Comment:
IMO we should make a breaking change and actually fix this props API. None of these props make a particular sense to me.

Reply:
Would just get rid of them all and then mixin `textDecoration` like `Clickable` does.

Reply:
Yeah good idea we can definitely replace noUnderline and underlineBehavior with it, but does textDecoration support hover and active colors?

Reply:
No but what would be the purpose of this component if you want to go against its styling? The goal of any of these components should be to make adhering to the style guide easy. Snowflakes can just do their own thing.

Reply:
There are a few instances in Volt where we would want to change the visited color as the links are being used more as buttons, but I guess could just change them to styled links.